app/models/manageiq/providers/openstack/helper_methods.rb

```diff
@@ -1,6 +1,10 @@
 module ManageIQ::Providers::Openstack::HelperMethods
   extend ActiveSupport::Concern
 
+  def openstack_proxy
+    VMDB::Util.http_proxy_uri(:openstack) || VMDB::Util.http_proxy_uri(:default)
```

Comment:
Isn't this basically just `ManageIQ::Providers::BaseManager#http_proxy_uri` ?  Are you not able to use that method where this is being called from?  (Not sure where `HelperMethods` is included)

Reply:
I think you can use the one from BaseManager when this is called by the ManagerMixin, maybe not from the legacy/openstack_handle code though

Reply:
I guess I can use the singleton method if nothing else. As for all the `to_s` calls, I don't think Excon takes a URI object, but let me check.